.github/workflows/parallel.yml

```diff
@@ -76,6 +77,10 @@ jobs:
         java-version: ${{ inputs.java-version }}
         distribution: ${{ inputs.jdk-distro }}
         cache: maven
+    - name: Set up Go 1.16
+      uses: actions/setup-go@v3
```

Comment:
We can set `if: ${{ matrix.profile ==  kubernetes}}` here, although it is not required.

Reply:
Could you raise a good first issue about this optimization?